### ISSUE TITLE
How do I input an open-vocabulary text 

### ISSUE THREAD

**Issue by 123lcy123** [Pointcept/OpenIns3D#9]
Hello, thank you for developing a good work, i am really interested in it. But I want to know how do I input an open-vocabulary text description like the example you gave?

**Comment by ZheningHuang** [comment]
Thank you for your interest in our work. We have not yet released the single detection testing code and plan to do so around early next month with the latest version of the paper. We will inform you in due course.

Best,
Zhening

**Comment by 123lcy123** [comment]
Thank you for your response and the update. I look forward to seeing the single detection testing code and the latest version of the paper.

**Comment by ZheningHuang** [comment]
We have just released a newer version of the code, which showcases how this is done under

[zero_shot_single_voc.py](https://github.com/Pointcept/OpenIns3D/blob/main/zero_shot_single_voc.py).

If you look into this code, you will see how the results in the teaser images are detected. Essentially, we designed a CLIP ranking and filtering system to effectively reduce false positive detections for single-vocabulary open-world inference.
Best,
Zhening
